scipy/fft/tests/mock_backend.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-import scipy.fft
+# import scipy.fft
```

Comment:
This is left over from an old version. You can remove it.

Reply:
Removed.